cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/OrderedTableDefinition.java

```diff
@@ -30,11 +30,12 @@
 /**
  * Defines Table data set
  */
-public class TableDefinition extends AbstractDatasetDefinition<Table, DatasetAdmin> {
+@Deprecated
+public class OrderedTableDefinition extends AbstractDatasetDefinition<Table, DatasetAdmin> {
 
-  private final DatasetDefinition<? extends OrderedTable, ?> tableDef;
+  private final DatasetDefinition<? extends Table, ?> tableDef;
 
-  public TableDefinition(String name, DatasetDefinition<? extends OrderedTable, ?> orderedTableDefinition) {
+  public OrderedTableDefinition(String name, DatasetDefinition<? extends Table, ?> orderedTableDefinition) {
```

Comment:
parameter naming is a bit confusing here. I guess it should be tableDefinition, not orderedTableDefinition?